boards/renesas/da1469x_dk_pro/da1469x_dk_pro-pinctrl.dtsi

```diff
@@ -64,12 +64,12 @@
 
 	spi_controller: spi_controller {
 		group1 {
-			pinmux = <SMARTBOND_PINMUX(SPI_CLK, 0, 22)>,
-				<SMARTBOND_PINMUX(SPI_DO, 0, 23)>;
+			pinmux = <SMARTBOND_PINMUX(SPI_CLK, 0, 21)>,
+				 <SMARTBOND_PINMUX(SPI_DO, 0, 25)>;
```

Comment:
The SPI MOSI signal on MikroBUS #1 socket should be routed to P0.26 and not to P0.25

Reply:
@ioannis-karachalios, Hmm, on boars that I have (`331-07-C` and `331-07-B`) it looks like it's P0_25, on the other hand on USB devkit it is P0_26.
Can you confirm that newest boards use P0_26? I will change it then.

Reply:
Oh, I see. I fall into the same trap as the labeling is not well printed on PCB. If you look at the [initial pin definition](https://github.com/zephyrproject-rtos/zephyr/pull/68426/commits/98b95b15096e25cf385d4602a90449ab5c654a25#diff-b3fbba5dd5b4bd56eebaee7fad01032a04a13c71b8ef35a74c971e6144c9902aL38) it should be P0.26. Here is a screenshot of the motherboard's schematic;
![mikrobus1_pins](https://github.com/zephyrproject-rtos/zephyr/assets/13284615/643ba7d2-8b85-4daf-84b1-71ed63cca3d7)